.github/workflows/build_docs.yml

```
@@ -12,20 +12,50 @@ on:
       - 'docs/**'
       - '.github/workflows/build_docs.yml'
   workflow_dispatch:
+  # Build docs if Animate, Goalie, Movement, or UM2N have been updated in the last 24h
+  schedule:
+    - cron: '0 0 * * *'
 
 concurrency:
   group: 'build_docs-${{ github.ref }}'
   cancel-in-progress: true
 
 jobs:
+  # Check if there have been commits on the main branch of other repos in the last 24h
+  check_updates:
+    name: Check for updates in Animate, Goalie, Movement, and UM2N
+    runs-on: ubuntu-latest
+    outputs:
+      updates_found: ${{ steps.check_repos.outputs.updates_found }}
+    steps:
+      - name: Set up environment
+        run: |
+          echo "REPOS=https://api.github.com/repos/mesh-adaptation/animate https://api.github.com/repos/mesh-adaptation/goalie https://api.github.com/repos/mesh-adaptation/movement https://api.github.com/repos/mesh-adaptation/UM2N" >> $GITHUB_ENV
+          echo "GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }}" >> $GITHUB_ENV
+
+      - name: Check for updates in repositories
+        id: check_repos
+        run: |
+          for repo in $REPOS; do
+            echo "Checking $repo"
+            commits=$(curl -L -H "Authorization: token $GITHUB_TOKEN" "$repo/commits?sha=main&since=$(date -d '24 hours ago' +'%Y-%m-%dT%H:%M:%SZ')")
+            echo $commits
+            if [ ${#commits} -gt 10 ]; then
+              echo "Found new commits in $repo"
+              echo "updates_found=true" >> $GITHUB_ENV
+            fi
+          done
```

I have a preference for capitals and curly braces when using Bash variables:
```suggestion
          for REPO in ${REPOS}; do
            echo "Checking ${REPO}"
            COMMITS=$(curl -L -H "Authorization: token $GITHUB_TOKEN" "${REPO}/commits?sha=main&since=$(date -d '24 hours ago' +'%Y-%m-%dT%H:%M:%SZ')")
            echo ${COMMITS}
            if [ ${#COMMITS} -gt 10 ]; then
              echo "Found new commits in ${REPO}"
              echo "updates_found=true" >> $GITHUB_ENV
            fi
          done
```

---

Done in a7b0453b6d40df17fe41ea09d4f7489991fe133a